backend/lcfs/web/api/notification/schema.py

```diff
@@ -1,11 +1,39 @@
+from datetime import datetime
 from typing import Any, Dict, List, Optional
 
 from lcfs.db.models.compliance.ComplianceReportStatus import ComplianceReportStatusEnum
 from lcfs.db.models.initiative_agreement.InitiativeAgreementStatus import (
     InitiativeAgreementStatusEnum,
 )
 from lcfs.db.models.transfer.TransferStatus import TransferStatusEnum
-from lcfs.web.api.base import BaseSchema, NotificationTypeEnum
+from lcfs.web.api.base import BaseSchema, NotificationTypeEnum, PaginationResponseSchema
+from pydantic import computed_field, model_validator
+
+
+class OrganizationSchema(BaseSchema):
```

Comment:
We should be re-using the one from Organization

Reply:
For same reason as below, don't want to expose too much information for this endpoint.

Reply:
changed as per suggestion.